bson/primitive/objectid.go

```diff
@@ -36,15 +36,26 @@ var processUnique = processUniqueBytes()
 
 // NewObjectID generates a new ObjectID.
 func NewObjectID() ObjectID {
+	return NewObjectIDFromTime(time.Now())
+}
+
+// NewObjectIDFromTime generates a new ObjectID based on the given time.
+func NewObjectIDFromTime(generationTime time.Time) ObjectID {
```

Comment:
Let's change this to `NewObjectIDFromTimestamp`, to match the requested change from `GenerationTime` to `Timestamp`.

Reply:
Renamed